src/setup/argparser.py

```diff
@@ -50,6 +58,7 @@ def parse_args() -> argparse.ArgumentParser:
                         type=int,
                         default=100
                         )
+    # TODO: flag --maker must be a required flag
```

Comment:
This needs to be adressed when running on CLI

Reply:
As a matter of fact, many flags should be improved because many features is needed, such as [Argument Groups](https://docs.python.org/3/library/argparse.html#argument-groups), [Mutual Exclusion](https://docs.python.org/3/library/argparse.html#mutual-exclusion) and most importantly [sub-commands](https://docs.python.org/3/library/argparse.html#sub-commands)

Reply:
Addressed in #26